config/default/kustomization.yaml

```diff
@@ -26,124 +21,7 @@ resources:
 # [PROMETHEUS] To enable prometheus monitor, uncomment all sections with 'PROMETHEUS'.
 #- ../prometheus
 
-secretGenerator:
-  - name: token
-    envs:
-    - .env.linode
-
-patchesStrategicMerge:
-# Protect the /metrics endpoint by putting it behind auth.
-# If you want your controller-manager to expose the /metrics
-# endpoint w/o any authn/z, please comment the following line.
-- manager_auth_proxy_patch.yaml
-
-
-
-# [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in
```

Comment:
should this webhook/certmanager stuff stay in here in case we want to use it in the future?

Reply:
I'd prefer to add that when we need it but I can add back if that's preferable